redis-gin-mongo/k8s/templates/db_worker1.yml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    velocity.tech.v1/id: db-worker1
+    velocity.tech.v1/dependsOn: redis, mongo1
+  name: db-worker1
+  labels:
+    app: db-worker1
+spec:
+  selector:
+    matchLabels:
+      api: db-worker1
+  replicas: 3
```

Comment:
pay attention that redis channels is working as "pub/sub" meaning that all workers will receive the same message and operate the same work.